edward/util.py

```diff
@@ -133,81 +168,152 @@ def kl_multivariate_normal(loc_one, scale_one, loc_two=0, scale_two=1):
             1.0 + 2.0 * tf.log(scale_two) - 2.0 * tf.log(scale_one), 1)
 
 def log_sum_exp(x):
-    """
-    Computes the log_sum_exp of the elements in x.
+    """Compute the ``log_sum_exp`` of the elements in x.
 
-    Works for x with
+    Parameters
+    ----------
+    x : tf.Tensor
+        vector or matrix with second dimension 1
         shape=TensorShape([Dimension(N)])
         shape=TensorShape([Dimension(N), Dimension(1)])
 
-    Not tested for anything beyond that.
+    Returns
+    -------
+    tf.Tensor
+        scalar if vector input, vector if matrix tensor input
     """
     x_max = tf.reduce_max(x)
     return tf.add(x_max, tf.log(tf.reduce_sum(tf.exp(tf.sub(x, x_max)))))
 
 def logit(x):
-    """log(x / (1 - x))"""
+    """Evaluates :math:`\log(x / (1 - x))` elementwise. 
```

Comment:
In my doc, I've personally stuck with active imperative, e.g., "Evaluate [...] elementwise."